gds_write/BUILD

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# GDS write tool package.
+
+load("@rules_hdl_pip_deps//:requirements.bzl", "requirement")
+load("@rules_python//python:defs.bzl", "py_binary")
+
+py_binary(
```

Comment:
should we have some `py_test` too?

Reply:
[Here](https://github.com/antmicro/bazel_rules_hdl/blob/gds-write/synthesis/tests/BUILD#L47-L53) we test the rule that uses this `py_binary`. If that is not enough I can add `py_test` but I think it would not be much different compared to the linked test. Please let me know what do you think.